transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__activity.sql

```diff
@@ -0,0 +1,73 @@
+
+
+with source as (
+    select * from {{ source('mm_telemetry_prod', 'activity') }}
+),
+renamed as (
+
+    select
+        -- Common event columns
+        id               as event_id
+        , event          as event_table
+        , event_text     as event_name
+        , user_id        as server_id
+        , received_at    as received_at
+        , timestamp      as timestamp
+
+        -- Server info
+        , coalesce(context_traits_installationid,  context_traits_installation_id) as installation_id
+        , anonymous_id
+        , context_ip as server_ip
+
+        -- Channel metrics
+        , public_channels as count_public_channels
+        , public_channels_deleted as count_deleted_public_channels
+        , private_channels as count_private_channels
+        , private_channels_deleted as count_deleted_private_channels
+        , bot_posts_previous_day as count_bot_posts_previous_day
+        , posts_previous_day as count_posts_previous_day
+        , teams as count_teams
+        , slash_commands as count_slash_commands
+        , direct_message_channels as count_direct_message_channels
+        , posts as count_posts
+        , incoming_webhooks as count_incoming_webhooks
+        , outgoing_webhooks as count_outgoing_webhooks
+
+        -- User & account metrics
+        , active_users_daily as daily_active_users
```

Comment:
maybe to open a bug to investigate at a certain moment why `active_users_daily` is populated and `daily_active_users` not

Reply:
There were only ~ 800 records (out of 100m+) that had a value in `daily_active_users`. The [source code of mattermost server](https://github.com/mattermost/mattermost/blob/master/server/platform/services/telemetry/telemetry.go#L379) is using `active_users_daily`. I even checked versions back to 4.x and the code seems to be using this alias since then. My assumptions is that it's either a test/dev server, or part of some pen test. Not sure if it's worth spending time to investigate this.

Reply:
That's fine - then maybe we should just add `daily_active_users` to a "to remove/not ingest" list to clean-up at a certain point- to avoid extreme sparseness/empty columns.

Reply:
I'm kind of documenting these things by leaving comments on the staging models about my investigation on these columns (here's an [example](https://github.com/mattermost/mattermost-data-warehouse/pull/1329/files#diff-ad95894abfb0112a643b913535b5b52076d7b7a0970400d1f58498d35f034f52R53-R55)). Do you think we need something more?

Reply:
Not really, as long as we can easily do a scrub to find them when the time comes = ).